packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -33,39 +33,97 @@ import {
 export async function runBrokenLinks(
   options: BrokenLinksResultV1_BrokenLinkCheckerOptions
 ): Promise<SyntheticResult> {
-  // START - to resolve warnings while under development
-  checkStatusPassing({ status_value: 200 } as ResponseStatusCode, 200);
-
-  const browser = await puppeteer.launch({ headless: 'new' });
-  const page = await browser.newPage();
-  await checkLink(
-    page,
-    {} as LinkIntermediate,
-    {} as BrokenLinksResultV1_BrokenLinkCheckerOptions
-  );
-  // END - to resolve warnings  while under development
-
   // options object modified directly
   setDefaultOptions(options);
 
-  // PSEUDOCODE
-
-  // create puppeteer.Browser
-  // create puppeteer.Page & navigate to origin_url, w/ origin specific settings
+  // create Browser & origin page then navigate to origin_url, w/ origin
+  // specific settings
+  const browser = await puppeteer.launch({ headless: 'new' });
+  const origin_page = await browser.newPage();
+  if (options.wait_for_selector)
```

Comment:
nit: new line between instantiation and logic

Reply:
done.